### ISSUE TITLE
Downgrading PMML-4_4 to PMML-4_3

### ISSUE THREAD

**Issue by IvanRoblesMunoz** [jpmml/jpmml-model#25]
Hi,

I have made a lightGBM model and converted it to a PMML using the jpmml-lightgbm library. 
The output file contains <PMML xmlns="http://www.dmg.org/PMML-4_4" xmlns:data="http://jpmml.org/jpmml-model/InlineTable" version="4.3">. 

The issue with this is that I was trying to import this model file into a software that only accepts "http://www.dmg.org/PMML-4_3" versions or lower. I managed to use the org.jpmml.model.example.CopyExample tool but I can only get this to create a file with the latest PMML version.
I was wondering if there is a way to downgrade PMML version.

Kind regards,
Ivan Robles

**Comment by vruusmann** [comment]
> I was wondering if there is a way to downgrade PMML version.

All JPMML conversion libraries are producing PMML schema version 4.4 documents for about two months now. There's a relevant announcement in the JPMML mailing list: https://groups.google.com/forum/#!topic/jpmml/9sVQSiW52-o

I have no interest/resources in maintaining multiple versions of conversion libraries (one for PMML schema version 4.4, another one for 4.3, etc). If that's a priority for you, feel free to build such a PMML schema version transformer yourself - the JPMML-Model library is a very good starting point for this.

TLDR: Use the latest version from the JPMML-LightGBM 1.2.X development branch:
https://github.com/jpmml/jpmml-lightgbm/releases/tag/1.2.14
